source/extensions/extensions_build_config.bzl

```diff
@@ -80,9 +80,8 @@ EXTENSIONS = {
     "envoy.filters.network.echo":                       "//source/extensions/filters/network/echo:config",
     "envoy.filters.network.ext_authz":                  "//source/extensions/filters/network/ext_authz:config",
     "envoy.filters.network.http_connection_manager":    "//source/extensions/filters/network/http_connection_manager:config",
-    # NOTE: Kafka filter does not have a proper filter implemented right now. We are referencing to
-    #       codec implementation that is going to be used by the filter.
-    "envoy.filters.network.kafka":                      "//source/extensions/filters/network/kafka:kafka_request_codec_lib",
+    # Not implemented yet
+    #"envoy.filters.network.kafka":                      "//source/extensions/filters/network/kafka:kafka_request_codec_lib",
```

Comment:
I think this may cause code/tests to not get compiled and run, same below. Can you check this?

Reply:
+1 the plan is to make incremental/small PRs (speaking about X-Ray tracer below). So, please make sure that's still possible given this change.

Reply:
I've added in a WiP status for filters that takes care of this, thanks for the reminder.